jena-arq/src/main/java/org/apache/jena/sparql/core/GraphView.java

```diff
@@ -165,6 +165,9 @@ public void performDelete( Triple t ) {
         Node o = t.getObject() ;
         dsg.delete(g, s, p, o) ;
     }
+    
+    // Subclasses may wish to provide graphBaseSize otherwise GraphBase uses find()  
```

Comment:
Maybe this is just where I like to find things, but I'd rather `@Override protected int graphBaseSize() { return super.graphBaseSize(); }` with this as a Javadoc comment.

Reply:
Done.